pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -375,6 +383,11 @@ func (r *ReconcileClusterDeployment) Reconcile(request reconcile.Request) (recon
 			if err != nil {
 				cdLog.Errorf("error creating job: %v", err)
 				return reconcile.Result{}, err
+			} else {
+				cdLog.Infof("CREATED INSTALL JOB IN:%s", job.Status.CompletionTime)
+				kickstartDuration := cd.CreationTimestamp.Sub(existingJob.Status.StartTime.Time)
```

Comment:
Should be something like: kickstartDuration := time.Since(cd.CreationTimestamp.Time). We don't care about when the job started rather what time is it right now.